src/autowiring/marshaller.h

```diff
@@ -171,104 +153,66 @@ namespace autowiring {
   };
 
   template<typename T>
+  struct float_converter;
+
+#ifdef MSVC
```

Comment:
Unfortunately, snprintf is undefined in MSVC2013 and 2015, only having been added in VC "15"